ui/packages/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js

```diff
@@ -123,6 +140,18 @@ module.exports = Marionette.LayoutView.extend({
         this.uploadBatchModel.clear();
     },
     startUpload: function() {
+        this.triggerMethod('ingestDetails:before:start');
+        for (var i = 0; i < properties.requiredAttributes.length; i++) {
+            var attrName = properties.requiredAttributes[i];
```

Comment:
❓ Maybe we should gather the required attributes before we announce, so we don't spam the user with multiple errors?

Reply:
Good idea. I'll change it